quickstart-sda.md

```diff
@@ -0,0 +1,240 @@
+
+```shell
+crypt4gh encrypt -f <file-to-encrypt> -p <sda-c4gh-public-key>
+```
+
+
+### Uploading data
+
+Users can upload data to the SDA by transferring them directly to the archive's `s3inbox` with an S3 client tool such as [`s3cmd`](https://s3tools.org/s3cmd):
+```
+
+```shell
+s3cmd -c s3cmd.conf put <path-to-file.c4gh> s3://<username>/<target-path-to-file.c4gh>
+```
+
+where `s3cmd.conf` is a configuration file with the following content:
+```ini
+[default]
+access_key = <USER_NAME>
+secret_key = <USER_NAME>
+access_token=<JW_TOKEN>
+check_ssl_certificate = False
+check_ssl_hostname = False
+encoding = UTF-8
+encrypt = False
+guess_mime_type = True
+host_base = <S3_INBOX_DOMAIN_NAME>
+host_bucket = <S3_INBOX_DOMAIN_NAME>
+human_readable_sizes = true
+multipart_chunk_size_mb = 50
+use_https = True
+socket_timeout = 30
+```
+
+### The sda-cli tool
+
+Instead of the tools above, users are **encouraged** to use [`sda-cli`](https://github.com/NBISweden/sda-cli), which is a tool specifically developed to perform all common SDA user-related tasks in a convenient and unified manner. It is recommended to use precompiled executables for `sda-cli` which can be found at https://github.com/NBISweden/sda-cli/releases
+
+To start using the tool run:
+
+```shell
+./sda-cli help
+```
+
+#### Examples of common usage
+
+- Encrypt and upload a file to the SDA in one go:
+
+```shell
+./sda-cli upload -config s3cmd.conf --encrypt-with-key <sda-c4gh-public-key> <unencrypted_file_to_upload>
+```
+
+- Encrypt and upload a whole folder recursively to a specified path, which can be different from the source, in one go:
+
+```shell
+./sda-cli upload -config s3cmd.conf --encrypt-with-key <sda-c4gh-public-key> -r <folder_1_to_upload> -targetDir <upload_folder>
+```
+
+- List all uploaded files in the user's bucket recursively:
+
+```shell
+./sda-cli list -config s3cmd.conf
+```
+For detailed documentation on the tool's capabilities and usage please refer [here](https://github.com/NBISweden/sda-cli#usage).
+
+### Downloading data
+
+Users can directly download data from the SDA via `sda-download`, for more details see the service's [api reference](https://github.com/neicnordic/sda-download/blob/main/docs/API.md) and the examples [here](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces). In short, given a valid token, `$token`,  a user can download the file with file ID, `$fileID` by issuing the following command:
+
+```shell
+curl --cacert <path-to-certificate-file> -H "Authorization: Bearer $token" https://<sda-download_DOMAIN_NAME>/files/$fileID -o <output-filename>
+```
+
+### Data access permissions
+
+In order for a user to access a file, permission to access the dataset that the file belongs to is needed. This is granted through [REMS](https://github.com/CSCfi/rems) in the form of `GA4GH` visas. For details see [starter-kit documentation on REMS](https://github.com/GenomicDataInfrastructure/starter-kit-rems) and the links therein.
```

Comment:
Maybe it's worth mentioning that this is not related to the OIDC and it is not possible to add permissions with the current setup

Reply:
Makes sense. but now with the addition of  `LS-AAI-mock` instead of our old `oidc` this is possible.